cloud-build/README.MD

```diff
@@ -0,0 +1,12 @@
+# Continuous Integration via Cloud Build
+
+Cloud Build is configured in our GCP project (google.com:cloud-dataproc-ci) to trigger whenever there are PRs from a fork to the main repository. We are using GitHub App Triggers Alpha to integrate with GitHub.
```

Comment:
I think we should remove "google.com:cloud-dataproc-ci", that is sensitive information.